src/doc/complement-design-faq.md

```diff
@@ -174,3 +174,10 @@ bindings.
 See also [a long thread][alt] on renaming `let mut` to `var`.
 
 [alt]: https://mail.mozilla.org/pipermail/rust-dev/2014-January/008319.html
+
+## Why no `--x` or `x++`?
+
+Pre- and post- increment, while convenient, are also fairly complex. They
```

Comment:
yeah, I felt slightly awkward about it, but it felt more consistent. Maybe I should just re-word and write 'increment' twice